docs/core/porting/cppcli.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Tools for porting to .NET Core
+description: Learn about porting C++/CLI projects to .NET Core
+author: mjrousos
+ms.author: 
+ms.date: 12/12/2019
+---
+
+# How to port a C++/CLI project to .NET Core
+
+Beginning with .NET Core 3.1 and Visual Studio 2019 16.4, C++/CLI projects can be updated to target .NET Core. This allows Windows desktop applications which may use C++/CLI projects as an interop layer between managed and native code to be ported to .NET Core. This article describes how to port C++/CLI projects from .NET Framework to .NET Core 3.1.
+
+## C++/CLI .NET Core limitations
+
+There are some important limitations to porting C++/CLI projects to .NET Core compared to C# or VB projects:
+
+* C++/CLI support for .NET Core is Windows only.
+* C++/CLI projects cannot target .NET Standard, only .NET Core (or, of course, .NET Framework).
+* C++/CLI projects do not support the new SDK-style project file format. Instead, even when targeting .NET Core, C++/CLI projects use the existing vcxproj file format.
+  * Because of this, C++/CLI projects cannot multitarget multiple .NET platforms. If you need to build a C++/CLI project for both .NET Framework and .NET Core, use separate project files.
+* .NET Core does not support /clr:pure or /clr:safe compilation, only the new /clr:netcore option (which is equivalent to /clr for .NET Framework).
```

Comment:
```suggestion
* .NET Core does not support `/clr:pure` or `/clr:safe` compilation, only the new `/clr:netcore` option (which is equivalent to `/clr` for .NET Framework).
```

Reply:
I'm not sure if the `-` can be used here instead of `/`.
See https://github.com/dotnet/docs/issues/9958#issuecomment-460437418.

Reply:
Both `-` and `/` [work ](https://docs.microsoft.com/en-us/cpp/build/reference/compiler-options?view=vs-2019) for cl.exe and link.exe. I'll switch to `-`.